src/cli/commands/__tests__/lint.test.ts

```diff
@@ -218,7 +218,7 @@ describe('lint', () => {
           expect(ctx.stdout).toContain(
             '46:24  warning  operation-description   Operation `description` must be present and non-empty string',
           );
-          expect(ctx.stdout).toContain('22 problems (0 errors, 22 warnings, 0 infos)');
+          expect(ctx.stdout).toContain('28 problems (0 errors, 28 warnings, 0 infos)');
```

Comment:
Why did this go up?

Reply:
It's because of the fix included here https://github.com/stoplightio/spectral/pull/336.
Basically, we couldn't evaluate oas-schema rue properly in some cases, therefore we didn't 
report all errors/warnings
The change is unrelated to this PR, I simply included the code since I worked on that at the same time.
The code is already merged, so this code is no longer present on this branch.
Is that clear now?

Reply:
If you revert the change introduced in 336, you will get 22 problems again.